build/findbugs/findbugs-exclude.xml

```diff
@@ -12,7 +12,12 @@
     <Class name="alluxio.yarn.ApplicationMaster"/>
     <Bug code="VO"/>
   </Match>
-
+  
+  <Match>
+    <Package name="alluxio.kafka.connect"/>
+    <Bug code="SF"/>
```

Comment:
In writeRecord method of AlluxioTopicPartitionWriter class , the switch-case statement is used to write data to Alluxio file. The implementation in each case corresponds to the current state. When the current state has been processed, the next state begins to be processed. So ,I didn't add beak statement in the end of case processing. But with this method , the build can not be passed . So , I add SF code for kafka-connect in findbugs-exculde.xml.  
Sorry , I didn't leave comments here before.